renovate.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Are there instructions we should add to the `README` so folks know how to get this working in their project repo? For the `vulnerabilityAlerts` config, it [sounds like](https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts) a project needs to:
 
> enable the [Dependency graph](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#enabling-the-dependency-graph), and [Dependabot alerts](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-security-and-analysis-settings-for-your-repository).

Reply:
Good idea. I was confused about this because as you can see in the PR, renovate surfaces 3 security alerts, which as of now dependabot hasn't surfaced for this repo that I know of (maybe we closed them but I don't see at a quick glance 👁️ ) 

I think I still don't have a good picture of what we need to *turn off* for dependabot and want we need to *enable* for the security alerts. 

Anyways, I may add a short comment to the docs about this for now but hopefully we can update once we know *exactly* what the deal is! I will poke around / research a bit more too.

Reply:
Added a ticket for this! https://github.com/navapbc/template-application-nextjs/issues/46